### ISSUE TITLE
slicker brought in an unneeded import

### ISSUE THREAD

**Issue by davidflanagan** [Khan/slicker#2]
With webapp in this state: https://github.com/Khan/webapp/commit/aa39930c84ad17154b943b2c917105957248fb84

I ran this command:

$ slicker.py content_editing.handlers.TutorialNavData content_render.handlers

It copied this line into content_render/handlers.py:

from content_editing.api import articles

That line is unused and it seems like a bug that it got added.

**Comment by benjaminjkraft** [comment]
I'll try to look at this if I get some spare time this week, I'm pretty curious about it.

**Comment by benjaminjkraft** [comment]
Ah, I see what's going on here.

The move-source file `content_editing/handlers.py` has the import `from content_editing.api import articles`, which is used, but additionally, the moved class `TutorialNavData` has a method with a local variable `articles`.  Because we don't try very hard to understand what is a local variable, we think that this is a reference to the module.  So we copy the import when we move the class.

I'll think about whether there's a good way to do this, short of handling local variables more carefully, which is a known TODO but quite hard.

**Comment by benjaminjkraft** [comment]
Yeah, I don't see how to do this without tracking variables and scopes, which I've now opened as #19.
